### PR TITLE
Update issue templates for new labels

### LINKED ISSUE #9718: Update page templates for new labels
As in https://github.com/dart-lang/site-www/issues/5268, update the issue template files to leverage the new labels.

### PR DESCRIPTION
**Pull request by atsansone**
Fixes #9718

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/4_infrastructure.yml [hunk 1]
```diff
@@ -1,26 +1,36 @@
 name: Site infrastructure issue
 description: |
-    Report a problem with the build, repo, or something else that's a problem for people who work on flutter.dev
-labels: [infrastructure]
+    Report a problem with the build, repo, or tooling for dart.dev
```
Comment (parlough) [inline .github/ISSUE_TEMPLATE/4_infrastructure.yml:3]:
```suggestion
    Report a problem with the build, repo, or tooling for docs.flutter.dev
```

File: .github/ISSUE_TEMPLATE/4_infrastructure.yml [hunk 2]
```diff
@@ -1,26 +1,36 @@
...
+        - Linux
+        - ChromeOS
   - type: textarea
     id: additional-info
     attributes:
       label: Additional context
       placeholder: Helpful examples, screenshots, or descriptions go here.
     validations:
-      required: false
+      required: false
```
Comment (parlough) [inline .github/ISSUE_TEMPLATE/4_infrastructure.yml:36]:
```suggestion
      required: false

```

**Review by parlough — COMMENTED** [review]
Thanks for updating these!

A few minor fixes needed after copying over from dart.dev:

File: .github/ISSUE_TEMPLATE/2_topic_request.yml [hunk 3]
```diff
@@ -1,18 +1,26 @@
...
+description: What information should be on docs.flutter.dev but isn't covered well (or at all)?
+labels: [co.request, from.user]
 body:
   - type: textarea
     id: problem
     attributes:
-      label: What information is missing?
-      placeholder: A clear and concise description of what you expected to find on flutter.dev, but didn't.
+      label: What information needs to be added?
+      placeholder: Provide a clear and concise description of what you expected to find on dart.dev, but didn't.
```
Comment (parlough) [inline .github/ISSUE_TEMPLATE/2_topic_request.yml:9]:
```suggestion
      placeholder: Provide a clear and concise description of what you expected to find on docs.flutter.dev, but didn't.
```

**Review by parlough — APPROVED** [review]
Thanks for making those updates! Looks good to me :)

**Review by domesticmouse — APPROVED** [review]

**Pull request merged**